droidlet/lowlevel/locobot/remote/slam_pkg/slam.py

```diff
@@ -90,16 +96,33 @@ def __init__(
         self.vis = vis
         self.save_vis = save_vis
         self.save_folder = save_folder
+        print(f"save_vis {save_vis}, save_folder {save_folder}")
```

Comment:
remove or make it a logging call, with a better message

Reply:
improved the message but keeping it as a print call for now since these get output on the docker stdout, where all other calls are also print calls for now. Will clean all the print calls up by adding a stream handler to the logger and using that at a later time.